lib/view_component/test_helpers.rb

```diff
@@ -2,6 +2,8 @@
 
 module ViewComponent
   module TestHelpers
+    extend ActiveSupport::Concern
```

Comment:
I'm curious- What necessitated this change?

Reply:
Whoops, this was leftover from when `controller_class` was available as a top-level macro method. Removed!